### ISSUE TITLE
Unable to run project after upgrading to Preview 5

### ISSUE THREAD

**Issue by efonsecab** [dotnet/aspire#3631]
I recently tried to update to Preview 5, and the host is unable to start.

These are the messages in the debug window

Aspire.Hosting.DistributedApplication: Information: Distributed application started. Press Ctrl+C to shut down.
Aspire.Hosting.Dcp.dcpctrl.ExecutableReconciler: Error: run session could not be started	{"Executable": {"name":"fairplaytube-h36gkm0"}, "Reconciliation": 7, "Status": "400 Bad Request", "Body": ""}

Aspire.Hosting.Dcp.dcpctrl.ExecutableReconciler: Error: run session could not be started	{"Executable": {"name":"fairplaytubevideoindexing-8s8psn8"}, "Reconciliation": 8, "Status": "400 Bad Request", "Body": ""}

Aspire.Hosting.Dcp.dcpctrl.ExecutableReconciler: Error: run session could not be started	{"Executable": {"name":"fairplayadminportal-0jtsdk8"}, "Reconciliation": 9, "Status": "400 Bad Request", "Body": ""}

'FairPlayCombinedSln.AppHost.exe' (CoreCLR: clrhost): Loaded 'Anonymously Hosted DynamicMethods Assembly'. 
'FairPlayCombinedSln.AppHost.exe' (CoreCLR: clrhost): Loaded 'C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App\9.0.0-preview.3.24172.13\Microsoft.AspNetCore.Metadata.dll'. 
'FairPlayCombinedSln.AppHost.exe' (CoreCLR: clrhost): Loaded 'C:\Program Files\dotnet\shared\Microsoft.NETCore.App\9.0.0-preview.3.24172.9\System.Linq.Expressions.dll'. 
'FairPlayCombinedSln.AppHost.exe' (CoreCLR: clrhost): Loaded 'C:\Program Files\dotnet\shared\Microsoft.NETCore.App\9.0.0-preview.3.24172.9\System.Runtime.CompilerServices.Unsafe.dll'. 
Exception thrown: 'System.OperationCanceledException' in System.Private.CoreLib.dll
Exception thrown: 'System.OperationCanceledException' in System.Private.CoreLib.dll
Exception thrown: 'System.OperationCanceledException' in System.Private.CoreLib.dll
Exception thrown: 'System.OperationCanceledException' in System.Private.CoreLib.dll
Exception thrown: 'System.OperationCanceledException' in System.Private.CoreLib.dll
Exception thrown: 'System.OperationCanceledException' in System.Private.CoreLib.dll
Exception thrown: 'System.OperationCanceledException' in System.Private.CoreLib.dll
Exception thrown: 'System.OperationCanceledException' in System.Private.CoreLib.dll
The thread '[Thread Destroyed]' (20216) has exited with code 0 (0x0).
The thread '[Thread Destroyed]' (29284) has exited with code 0 (0x0).
The thread '[Thread Destroyed]' (22716) has exited with code 0 (0x0).
The thread '[Thread Destroyed]' (14264) has exited with code 0 (0x0).
The thread '[Thread Destroyed]' (17860) has exited with code 0 (0x0).
The thread '[Thread Destroyed]' (2076) has exited with code 0 (0x0).
The program '[16584] FairPlayCombinedSln.AppHost.exe' has exited with code 4294967295 (0xffffffff).

**Comment by davidfowl** [comment]
Can you share more information, show more output (the console output shows the version of the Aspire.Hosting runtime in the beginning). Can you also show the output of `dotnet workload list`?

**Comment by efonsecab** [comment]
I was able to solve this, not exactly sure how, I had to do lots of things, latest I could that fixed the issue in several computers was moving to Preview 4.
Run the command lines ( I'm using .NET 9 )
dotnet workload update --source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet9/nuget/v3/index.json --skip-sign-check
Run the command line while inside the AppHost project directory.
dotnet workload restore --skip-sign-check

For .NET 9 there is a bug for the Content-Security header, so I additionally included a global.json to force the specific sdk to use.

https://github.com/pticostaricags/FairPlayCombined/tree/dev-aspire-prev4
